synapse/handlers/device.py

```diff
@@ -1196,10 +1161,9 @@ async def _maybe_retry_device_resync(self) -> None:
             for user_id in need_resync:
                 try:
                     # Try to resync the current user's devices list.
```

Comment:
We should probably add a TODO comment here that says to resync multiple users at once...